R/module_teal.R

```diff
@@ -118,8 +73,40 @@ srv_teal <- function(id, data, modules, filter = teal_slices()) {
       shinyjs::showLog()
     }
 
-    # `JavaScript` code
-    run_js_files(files = "init.js")
+    output$main_teal_ui <- renderUI({
```

Comment:
Given that data module is a modal it is not needed so much to keep it in the server. IMO this can be moved to the UI